source/style-guide/writing/write-to-the-user.txt

```diff
@@ -1,13 +1,19 @@
 .. _write-to-the-user:
+.. _sg-second-person:
+.. _sg-imperative-mood:
 
 =====================================
 Use Second Person and Imperative Mood
 =====================================
 
+.. default-domain:: mongodb
+
 .. include:: /includes/styles/corrections.rst
 
-Users are more engaged with content when it talks to them directly. You
-talk to users directly by using *second person*, addressing the user as
+Write to the User
+-----------------
+
+Users are more engaged with content when it talks to them directly. Use *second person* to talk to users directly, addressing the user as
```

Comment:
This is especially important in instructionals / procedurals (w/ imperative). I'd recommend, even, calling that out specifically:

While second person is recommended for general use throughout our documentation, it is especially important for procedural documentation where you are guiding a user through a step-by step process. "After you've installed the software, configure authentication" or "edit your config file to contain the following options".

Reply:
It is in the first bullet point.